src/ui/src/data/matchQueue.ts

```diff
@@ -1,35 +1,21 @@
-import { Record as RLRecord } from "@/models/record";
+import { RecordMatch } from "@/models/recordMatch";
+import { API_URL } from "@/utils/constants";
+import { deserializeRecordMatch } from "@/utils/deserializers";
+import { AppError } from "@/utils/errors";
 
-export async function getUnmatchedRecords(): Promise<RLRecord[]> {
-  // return (await fetch(`${API_URL}/demo/record`)).json();
-  // return [];
-  // return Promise.reject(new Error("oooops"));
-  return Promise.resolve([
-    {
-      id: 123,
-      first_name: "John",
-      last_name: "Doe",
-      birth_date: new Date("01/07/1993"),
-      received_on: new Date("04/22/2021"),
-      data_stream: {
-        system: "System system",
-        type: "ELR",
-      },
-      link_score: 0.9,
-      linked: null,
-    },
-    {
-      id: 124,
-      first_name: "Jane",
-      last_name: "Doe",
-      birth_date: new Date("01/07/1993"),
-      received_on: new Date("04/22/2021"),
-      data_stream: {
-        system: "System system",
-        type: "ELR",
-      },
-      link_score: 0.9,
-      linked: null,
-    },
-  ]);
+export async function getUnmatchedRecords(): Promise<RecordMatch[]> {
+  const response = await fetch(`${API_URL}/demo/record?status=pending`);
```

Comment:
If we use the `use server` directive that I suggested above, we can change the constant from `NEXT_PUBLIC_API_URL` to `RECORD_LINKER_API_URL`, which I think is clearer for the API endpoint.

Reply:
Same as above. For similar reasons we need to prefix all variables with `NEXT_PUBLIC` because they need to be injected at build time to become available to the static assets (js and css bundles).